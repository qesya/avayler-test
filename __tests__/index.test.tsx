import { render, screen, waitFor } from '@testing-library/react';
import HomePage, { getServerSideProps } from '../pages/index';
import '@testing-library/jest-dom'
import { Launch } from '../types/launch';
import { GetServerSidePropsContext } from 'next';

const mockResponse: Array<Launch> = [
  {
    "fairings": {
      "reused": false,
      "recovery_attempt": false,
      "recovered": false,
      "ships": []
    },
    "links": {
      "patch": {
        "small": "https://images2.imgbox.com/94/f2/NN6Ph45r_o.png",
        "large": "https://images2.imgbox.com/5b/02/QcxHUb5V_o.png"
      },
      "reddit": {
        "campaign": null,
        "launch": null,
        "media": null,
        "recovery": null
      },
      "flickr": {
        "small": [],
        "original": []
      },
      "presskit": null,
      "webcast": "https://www.youtube.com/watch?v=0a_00nJ_Y88",
      "youtube_id": "0a_00nJ_Y88",
      "article": "https://www.space.com/2196-spacex-inaugural-falcon-1-rocket-lost-launch.html",
      "wikipedia": "https://en.wikipedia.org/wiki/DemoSat"
    },
    "static_fire_date_utc": "2006-03-17T00:00:00.000Z",
    "static_fire_date_unix": 1142553600,
    "net": false,
    "window": 0,
    "rocket": "5e9d0d95eda69955f709d1eb",
    "success": false,
    "failures": [
      {
        "time": 33,
        "altitude": null,
        "reason": "merlin engine failure"
      }
    ],
    "details": "Engine failure at 33 seconds and loss of vehicle",
    "crew": [],
    "ships": [],
    "capsules": [],
    "payloads": [
      "5eb0e4b5b6c3bb0006eeb1e1"
    ],
    "launchpad": "5e9e4502f5090995de566f86",
    "flight_number": 1,
    "name": "FalconSat",
    "date_utc": "2006-03-24T22:30:00.000Z",
    "date_unix": 1143239400,
    "date_local": "2006-03-25T10:30:00+12:00",
    "date_precision": "hour",
    "upcoming": false,
    "cores": [
      {
        "core": "5e9e289df35918033d3b2623",
        "flight": 1,
        "gridfins": false,
        "legs": false,
        "reused": false,
        "landing_attempt": false,
        "landing_success": null,
        "landing_type": null,
        "landpad": null
      }
    ],
    "auto_update": true,
    "tbd": false,
    "launch_library_id": null,
    "id": "5eb87cd9ffd86e000604b32a"
  },
  {
    "fairings": {
      "reused": false,
      "recovery_attempt": false,
      "recovered": false,
      "ships": []
    },
    "links": {
      "patch": {
        "small": "https://images2.imgbox.com/f9/4a/ZboXReNb_o.png",
        "large": "https://images2.imgbox.com/80/a2/bkWotCIS_o.png"
      },
      "reddit": {
        "campaign": null,
        "launch": null,
        "media": null,
        "recovery": null
      },
      "flickr": {
        "small": [],
        "original": []
      },
      "presskit": null,
      "webcast": "https://www.youtube.com/watch?v=Lk4zQ2wP-Nc",
      "youtube_id": "Lk4zQ2wP-Nc",
      "article": "https://www.space.com/3590-spacex-falcon-1-rocket-fails-reach-orbit.html",
      "wikipedia": "https://en.wikipedia.org/wiki/DemoSat"
    },
    "static_fire_date_utc": '',
    "static_fire_date_unix": 0,
    "net": false,
    "window": 0,
    "rocket": "5e9d0d95eda69955f709d1eb",
    "success": false,
    "failures": [
      {
        "time": 301,
        "altitude": 289,
        "reason": "harmonic oscillation leading to premature engine shutdown"
      }
    ],
    "details": "Successful first stage burn and transition to second stage, maximum altitude 289 km, Premature engine shutdown at T+7 min 30 s, Failed to reach orbit, Failed to recover first stage",
    "crew": [],
    "ships": [],
    "capsules": [],
    "payloads": [
      "5eb0e4b6b6c3bb0006eeb1e2"
    ],
    "launchpad": "5e9e4502f5090995de566f86",
    "flight_number": 2,
    "name": "DemoSat",
    "date_utc": "2007-03-21T01:10:00.000Z",
    "date_unix": 1174439400,
    "date_local": "2007-03-21T13:10:00+12:00",
    "date_precision": "hour",
    "upcoming": false,
    "cores": [
      {
        "core": "5e9e289ef35918416a3b2624",
        "flight": 1,
        "gridfins": false,
        "legs": false,
        "reused": false,
        "landing_attempt": false,
        "landing_success": null,
        "landing_type": null,
        "landpad": null
      }
    ],
    "auto_update": true,
    "tbd": false,
    "launch_library_id": null,
    "id": "5eb87cdaffd86e000604b32b"
  },
  {
    "fairings": {
      "reused": false,
      "recovery_attempt": false,
      "recovered": false,
      "ships": []
    },
    "links": {
      "patch": {
        "small": "https://images2.imgbox.com/89/2a/bkI6LN0R_o.png",
        "large": "https://images2.imgbox.com/24/c3/9MKjvOdD_o.png"
      },
      "reddit": {
        "campaign": "https://www.reddit.com/r/spacex/comments/5dii6z",
        "launch": "https://www.reddit.com/r/spacex/comments/5nsaqm",
        "media": "https://www.reddit.com/r/spacex/comments/5nsico",
        "recovery": "https://www.reddit.com/r/spacex/comments/5oe9kk"
      },
      "flickr": {
        "small": [],
        "original": [
          "https://farm1.staticflickr.com/658/32394688795_55a9873ea7_o.jpg",
          "https://farm1.staticflickr.com/506/32394688095_a3339f3c6d_o.jpg",
          "https://farm1.staticflickr.com/745/32394687645_63ae2b4740_o.jpg",
          "https://farm1.staticflickr.com/318/31548291014_e3a30abca8_o.jpg",
          "https://farm1.staticflickr.com/670/32351549066_e9cffe8d2b_o.jpg",
          "https://farm6.staticflickr.com/5518/31579784413_83aeac560a_o.jpg",
          "https://farm6.staticflickr.com/5556/32312421135_22c197c156_o.jpg",
          "https://farm1.staticflickr.com/529/32312420015_5d2403a847_o.jpg",
          "https://farm1.staticflickr.com/435/32312417695_19c0e50c4b_o.jpg",
          "https://farm1.staticflickr.com/735/32312416415_b90892af0a_o.jpg",
          "https://farm1.staticflickr.com/293/32312415025_cae16d1994_o.jpg",
          "https://farm1.staticflickr.com/738/31467130724_92e02c9524_o.jpg",
          "https://farm1.staticflickr.com/464/31467130374_9f7a7d380e_o.jpg",
          "https://farm1.staticflickr.com/581/31467129424_bac77d594a_o.jpg",
          "https://farm1.staticflickr.com/380/32308163845_c1731a4b1f_o.jpg",
          "https://farm1.staticflickr.com/447/31450835954_72ed10a19e_o.jpg",
          "https://farm1.staticflickr.com/507/31450834974_b8a3f4aca5_o.jpg"
        ]
      },
      "presskit": "https://drive.google.com/open?id=0BwA3a65ef1OvZC1aU3FuMlQzalE",
      "webcast": "https://www.youtube.com/watch?v=7WimRhydggo",
      "youtube_id": "7WimRhydggo",
      "article": "https://spaceflightnow.com/2017/01/14/spacex-resumes-flights-with-on-target-launch-for-iridium/",
      "wikipedia": "https://en.wikipedia.org/wiki/Iridium_satellite_constellation#Next-generation_constellation"
    },
    "static_fire_date_utc": "2017-01-05T19:40:00.000Z",
    "static_fire_date_unix": 1483645200,
    "net": false,
    "window": 0,
    "rocket": "5e9d0d95eda69973a809d1ec",
    "success": true,
    "failures": [],
    "details": "Return-to-flight mission after the loss of Amos-6 in September 2016. Iridium NEXT will replace the original Iridium constellation, launched in the late 1990s. Each Falcon mission will carry 10 satellites, with a goal to complete deployment of the 66 plus 9 spare satellite constellation by mid 2018. The first two Iridium qualification units were supposed to ride a Dnepr rocket in April 2016 but were delayed, so Iridium decided to qualify the first batch of 10 satellites instead.",
    "crew": [],
    "ships": [
      "5ea6ed2f080df4000697c910",
      "5ea6ed30080df4000697c912",
      "5ea6ed30080df4000697c915"
    ],
    "capsules": [],
    "payloads": [
      "5eb0e4c2b6c3bb0006eeb208"
    ],
    "launchpad": "5e9e4502f509092b78566f87",
    "flight_number": 35,
    "name": "Iridium NEXT Mission 1",
    "date_utc": "2017-01-14T17:54:00.000Z",
    "date_unix": 1484416440,
    "date_local": "2017-01-14T10:54:00-07:00",
    "date_precision": "hour",
    "upcoming": false,
    "cores": [
      {
        "core": "5e9e28a3f359189e3a3b2645",
        "flight": 1,
        "gridfins": true,
        "legs": true,
        "reused": false,
        "landing_attempt": true,
        "landing_success": true,
        "landing_type": "ASDS",
        "landpad": "5e9e3033383ecbb9e534e7cc"
      }
    ],
    "auto_update": true,
    "tbd": false,
    "launch_library_id": null,
    "id": "5eb87cfdffd86e000604b34c"
  },
]

describe('Test HomePage', () => {
  it('Should renders HomePage', async () => {
    const component = render(<HomePage launches={mockResponse} />);
    expect(component).toMatchSnapshot();
  });

  it('Should renders a list of launches', async () => {
    render(<HomePage launches={mockResponse} />);
    expect(screen.getByText('DemoSat')).toBeInTheDocument();
    expect(screen.getByText('Iridium NEXT Mission 1')).toBeInTheDocument();
  });
})