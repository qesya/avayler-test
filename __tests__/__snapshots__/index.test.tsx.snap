// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test HomePage Should renders HomePage 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div>
        <nav
          class="fixed w-full z-30 py-6 bg-black px-10 md:px-0"
        >
          <div
            class="container mx-auto flex flex-row justify-between items-center"
          >
            <div>
              <h4
                class="font-Rajdhani text-white text-2xl"
                data-testid="Test-Header"
              >
                SpaceX
              </h4>
            </div>
            <div
              class="flex flex-row justify-center items-center"
            />
          </div>
        </nav>
        <section
          class="h-screen min-h-full bg-black"
        >
          <div
            class="container mx-auto flex flex-col h-full justify-around px-10 md:px-0"
          >
            <div
              class="flex flex-col mt-48"
            >
              <h1
                class="text-3xl md:text-6xl mt-16 font-AbrilFatface text-white md:w-4/12 leading-none"
              >
                What is SpaceX ?
              </h1>
              <p
                class="font-Rajdhani text-white text-base md:w-4/12 mt-10"
              >
                SpaceX is a private space company founded by Elon Musk in 2002 that designs, manufactures, and launches rockets and spacecraft.
              </p>
              <div>
                <button
                  class="bg-red-600 py-4 px-16 flex self-start mt-10 rounded-md"
                  data-testid="Test-Button"
                >
                  <p
                    class="font-AbrilFatface text-white hover:text-black w-full"
                    data-testid="Test-Title"
                  >
                    Discover Rockets
                  </p>
                </button>
              </div>
            </div>
            <div
              class="flex flex-col md:flex-row mt-16"
            >
              <div
                class="flex flex-row items-center"
              >
                <h4
                  class="text-white font-AbrilFatface text-3xl md:text-6xl mr-4"
                >
                  10
                </h4>
                <h4
                  class="text-white font-AbrilFatface text-3xl md:w-3/12 leading-none"
                >
                  Top Launches
                </h4>
              </div>
            </div>
            <img
              alt=""
              class="hidden md:flex md:absolute md:right-0 md:top-0 md:z-0 md:bottom-0 select-none"
              src="/images/planet.png"
            />
          </div>
        </section>
        <section
          class="py-20 bg-black"
          id="target"
        >
          <div
            class="container mx-auto flex flex-col h-full px-10 md:px-0"
          >
            <h1
              class="font-AbrilFatface text-4xl text-white mt-20"
            >
              List Of Launches
            </h1>
            <hr
              class="bg-white h-1 w-3/12 mt-6 mb-10"
            />
            <div
              class="flex flex-col md:flex-row justify-between my-10"
            >
              <div
                class="flex flex-row"
              >
                <h2
                  class="font-AbrilFatface text-5xl text-white mr-6"
                >
                  1
                </h2>
                <div
                  class="flex flex-col"
                >
                  <h2
                    class="font-AbrilFatface text-5xl text-white mb-4"
                  >
                    FalconSat
                  </h2>
                  <h4
                    class="font-Rajdhani text-white text-2xl"
                  >
                    ID : 
                    5eb87cd9ffd86e000604b32a
                  </h4>
                  <h4
                    class="font-Rajdhani text-white text-2xl"
                  >
                    Payloads : 
                    5eb0e4b5b6c3bb0006eeb1e1
                  </h4>
                  <h4
                    class="font-Rajdhani text-white text-2xl"
                  >
                    Date: 
                    Saturday, March 25, 2006 5:30 AM
                  </h4>
                  <h4
                    class="font-Rajdhani text-white text-2xl"
                  >
                    Cores : 
                    5e9e289df35918033d3b2623
                  </h4>
                  <h4
                    class="font-Rajdhani text-white text-2xl"
                  >
                    Status : 
                    FAILURE ðŸš©
                  </h4>
                  <h4
                    class="font-Rajdhani text-white text-2xl"
                  >
                    Reason of failure : 
                  </h4>
                  <div
                    class="flex-1 px-8"
                    data-testid="Test-Failure"
                  >
                    <p
                      class="font-Rajdhani text-white text-2xl"
                    >
                      ðŸ”´ Reason : 
                      merlin engine failure
                    </p>
                    <p
                      class="font-Rajdhani text-white text-2xl"
                    >
                      ðŸ”´ Altitude: 
                      -
                    </p>
                    <p
                      class="font-Rajdhani text-white text-2xl"
                    >
                      ðŸ”´ Time: 
                      33
                    </p>
                  </div>
                </div>
              </div>
              <img
                class="w-24 h-24"
                data-testid="Test-Image"
                src="https://images2.imgbox.com/94/f2/NN6Ph45r_o.png"
              />
            </div>
            <div
              class="flex flex-col md:flex-row justify-between my-10"
            >
              <div
                class="flex flex-row"
              >
                <h2
                  class="font-AbrilFatface text-5xl text-white mr-6"
                >
                  2
                </h2>
                <div
                  class="flex flex-col"
                >
                  <h2
                    class="font-AbrilFatface text-5xl text-white mb-4"
                  >
                    DemoSat
                  </h2>
                  <h4
                    class="font-Rajdhani text-white text-2xl"
                  >
                    ID : 
                    5eb87cdaffd86e000604b32b
                  </h4>
                  <h4
                    class="font-Rajdhani text-white text-2xl"
                  >
                    Payloads : 
                    5eb0e4b6b6c3bb0006eeb1e2
                  </h4>
                  <h4
                    class="font-Rajdhani text-white text-2xl"
                  >
                    Date: 
                    Wednesday, March 21, 2007 8:10 AM
                  </h4>
                  <h4
                    class="font-Rajdhani text-white text-2xl"
                  >
                    Cores : 
                    5e9e289ef35918416a3b2624
                  </h4>
                  <h4
                    class="font-Rajdhani text-white text-2xl"
                  >
                    Status : 
                    FAILURE ðŸš©
                  </h4>
                  <h4
                    class="font-Rajdhani text-white text-2xl"
                  >
                    Reason of failure : 
                  </h4>
                  <div
                    class="flex-1 px-8"
                    data-testid="Test-Failure"
                  >
                    <p
                      class="font-Rajdhani text-white text-2xl"
                    >
                      ðŸ”´ Reason : 
                      harmonic oscillation leading to premature engine shutdown
                    </p>
                    <p
                      class="font-Rajdhani text-white text-2xl"
                    >
                      ðŸ”´ Altitude: 
                      289
                    </p>
                    <p
                      class="font-Rajdhani text-white text-2xl"
                    >
                      ðŸ”´ Time: 
                      301
                    </p>
                  </div>
                </div>
              </div>
              <img
                class="w-24 h-24"
                data-testid="Test-Image"
                src="https://images2.imgbox.com/f9/4a/ZboXReNb_o.png"
              />
            </div>
            <div
              class="flex flex-col md:flex-row justify-between my-10"
            >
              <div
                class="flex flex-row"
              >
                <h2
                  class="font-AbrilFatface text-5xl text-white mr-6"
                >
                  3
                </h2>
                <div
                  class="flex flex-col"
                >
                  <h2
                    class="font-AbrilFatface text-5xl text-white mb-4"
                  >
                    Iridium NEXT Mission 1
                  </h2>
                  <h4
                    class="font-Rajdhani text-white text-2xl"
                  >
                    ID : 
                    5eb87cfdffd86e000604b34c
                  </h4>
                  <h4
                    class="font-Rajdhani text-white text-2xl"
                  >
                    Payloads : 
                    5eb0e4c2b6c3bb0006eeb208
                  </h4>
                  <h4
                    class="font-Rajdhani text-white text-2xl"
                  >
                    Date: 
                    Sunday, January 15, 2017 12:54 AM
                  </h4>
                  <h4
                    class="font-Rajdhani text-white text-2xl"
                  >
                    Cores : 
                    5e9e28a3f359189e3a3b2645
                  </h4>
                  <h4
                    class="font-Rajdhani text-white text-2xl"
                  >
                    Status : 
                    SUCCESS ðŸš€
                  </h4>
                </div>
              </div>
              <img
                class="w-24 h-24"
                data-testid="Test-Image"
                src="https://images2.imgbox.com/89/2a/bkI6LN0R_o.png"
              />
            </div>
          </div>
        </section>
        <div
          class="bg-black pt-20 pb-16"
        >
          <div
            class="container mx-auto flex flex-row justify-center"
          >
            <h3
              class="text-white text-3xl font-Rajdhani"
            >
              Pasquale Palena Â© 2023
            </h3>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div>
      <nav
        class="fixed w-full z-30 py-6 bg-black px-10 md:px-0"
      >
        <div
          class="container mx-auto flex flex-row justify-between items-center"
        >
          <div>
            <h4
              class="font-Rajdhani text-white text-2xl"
              data-testid="Test-Header"
            >
              SpaceX
            </h4>
          </div>
          <div
            class="flex flex-row justify-center items-center"
          />
        </div>
      </nav>
      <section
        class="h-screen min-h-full bg-black"
      >
        <div
          class="container mx-auto flex flex-col h-full justify-around px-10 md:px-0"
        >
          <div
            class="flex flex-col mt-48"
          >
            <h1
              class="text-3xl md:text-6xl mt-16 font-AbrilFatface text-white md:w-4/12 leading-none"
            >
              What is SpaceX ?
            </h1>
            <p
              class="font-Rajdhani text-white text-base md:w-4/12 mt-10"
            >
              SpaceX is a private space company founded by Elon Musk in 2002 that designs, manufactures, and launches rockets and spacecraft.
            </p>
            <div>
              <button
                class="bg-red-600 py-4 px-16 flex self-start mt-10 rounded-md"
                data-testid="Test-Button"
              >
                <p
                  class="font-AbrilFatface text-white hover:text-black w-full"
                  data-testid="Test-Title"
                >
                  Discover Rockets
                </p>
              </button>
            </div>
          </div>
          <div
            class="flex flex-col md:flex-row mt-16"
          >
            <div
              class="flex flex-row items-center"
            >
              <h4
                class="text-white font-AbrilFatface text-3xl md:text-6xl mr-4"
              >
                10
              </h4>
              <h4
                class="text-white font-AbrilFatface text-3xl md:w-3/12 leading-none"
              >
                Top Launches
              </h4>
            </div>
          </div>
          <img
            alt=""
            class="hidden md:flex md:absolute md:right-0 md:top-0 md:z-0 md:bottom-0 select-none"
            src="/images/planet.png"
          />
        </div>
      </section>
      <section
        class="py-20 bg-black"
        id="target"
      >
        <div
          class="container mx-auto flex flex-col h-full px-10 md:px-0"
        >
          <h1
            class="font-AbrilFatface text-4xl text-white mt-20"
          >
            List Of Launches
          </h1>
          <hr
            class="bg-white h-1 w-3/12 mt-6 mb-10"
          />
          <div
            class="flex flex-col md:flex-row justify-between my-10"
          >
            <div
              class="flex flex-row"
            >
              <h2
                class="font-AbrilFatface text-5xl text-white mr-6"
              >
                1
              </h2>
              <div
                class="flex flex-col"
              >
                <h2
                  class="font-AbrilFatface text-5xl text-white mb-4"
                >
                  FalconSat
                </h2>
                <h4
                  class="font-Rajdhani text-white text-2xl"
                >
                  ID : 
                  5eb87cd9ffd86e000604b32a
                </h4>
                <h4
                  class="font-Rajdhani text-white text-2xl"
                >
                  Payloads : 
                  5eb0e4b5b6c3bb0006eeb1e1
                </h4>
                <h4
                  class="font-Rajdhani text-white text-2xl"
                >
                  Date: 
                  Saturday, March 25, 2006 5:30 AM
                </h4>
                <h4
                  class="font-Rajdhani text-white text-2xl"
                >
                  Cores : 
                  5e9e289df35918033d3b2623
                </h4>
                <h4
                  class="font-Rajdhani text-white text-2xl"
                >
                  Status : 
                  FAILURE ðŸš©
                </h4>
                <h4
                  class="font-Rajdhani text-white text-2xl"
                >
                  Reason of failure : 
                </h4>
                <div
                  class="flex-1 px-8"
                  data-testid="Test-Failure"
                >
                  <p
                    class="font-Rajdhani text-white text-2xl"
                  >
                    ðŸ”´ Reason : 
                    merlin engine failure
                  </p>
                  <p
                    class="font-Rajdhani text-white text-2xl"
                  >
                    ðŸ”´ Altitude: 
                    -
                  </p>
                  <p
                    class="font-Rajdhani text-white text-2xl"
                  >
                    ðŸ”´ Time: 
                    33
                  </p>
                </div>
              </div>
            </div>
            <img
              class="w-24 h-24"
              data-testid="Test-Image"
              src="https://images2.imgbox.com/94/f2/NN6Ph45r_o.png"
            />
          </div>
          <div
            class="flex flex-col md:flex-row justify-between my-10"
          >
            <div
              class="flex flex-row"
            >
              <h2
                class="font-AbrilFatface text-5xl text-white mr-6"
              >
                2
              </h2>
              <div
                class="flex flex-col"
              >
                <h2
                  class="font-AbrilFatface text-5xl text-white mb-4"
                >
                  DemoSat
                </h2>
                <h4
                  class="font-Rajdhani text-white text-2xl"
                >
                  ID : 
                  5eb87cdaffd86e000604b32b
                </h4>
                <h4
                  class="font-Rajdhani text-white text-2xl"
                >
                  Payloads : 
                  5eb0e4b6b6c3bb0006eeb1e2
                </h4>
                <h4
                  class="font-Rajdhani text-white text-2xl"
                >
                  Date: 
                  Wednesday, March 21, 2007 8:10 AM
                </h4>
                <h4
                  class="font-Rajdhani text-white text-2xl"
                >
                  Cores : 
                  5e9e289ef35918416a3b2624
                </h4>
                <h4
                  class="font-Rajdhani text-white text-2xl"
                >
                  Status : 
                  FAILURE ðŸš©
                </h4>
                <h4
                  class="font-Rajdhani text-white text-2xl"
                >
                  Reason of failure : 
                </h4>
                <div
                  class="flex-1 px-8"
                  data-testid="Test-Failure"
                >
                  <p
                    class="font-Rajdhani text-white text-2xl"
                  >
                    ðŸ”´ Reason : 
                    harmonic oscillation leading to premature engine shutdown
                  </p>
                  <p
                    class="font-Rajdhani text-white text-2xl"
                  >
                    ðŸ”´ Altitude: 
                    289
                  </p>
                  <p
                    class="font-Rajdhani text-white text-2xl"
                  >
                    ðŸ”´ Time: 
                    301
                  </p>
                </div>
              </div>
            </div>
            <img
              class="w-24 h-24"
              data-testid="Test-Image"
              src="https://images2.imgbox.com/f9/4a/ZboXReNb_o.png"
            />
          </div>
          <div
            class="flex flex-col md:flex-row justify-between my-10"
          >
            <div
              class="flex flex-row"
            >
              <h2
                class="font-AbrilFatface text-5xl text-white mr-6"
              >
                3
              </h2>
              <div
                class="flex flex-col"
              >
                <h2
                  class="font-AbrilFatface text-5xl text-white mb-4"
                >
                  Iridium NEXT Mission 1
                </h2>
                <h4
                  class="font-Rajdhani text-white text-2xl"
                >
                  ID : 
                  5eb87cfdffd86e000604b34c
                </h4>
                <h4
                  class="font-Rajdhani text-white text-2xl"
                >
                  Payloads : 
                  5eb0e4c2b6c3bb0006eeb208
                </h4>
                <h4
                  class="font-Rajdhani text-white text-2xl"
                >
                  Date: 
                  Sunday, January 15, 2017 12:54 AM
                </h4>
                <h4
                  class="font-Rajdhani text-white text-2xl"
                >
                  Cores : 
                  5e9e28a3f359189e3a3b2645
                </h4>
                <h4
                  class="font-Rajdhani text-white text-2xl"
                >
                  Status : 
                  SUCCESS ðŸš€
                </h4>
              </div>
            </div>
            <img
              class="w-24 h-24"
              data-testid="Test-Image"
              src="https://images2.imgbox.com/89/2a/bkI6LN0R_o.png"
            />
          </div>
        </div>
      </section>
      <div
        class="bg-black pt-20 pb-16"
      >
        <div
          class="container mx-auto flex flex-row justify-center"
        >
          <h3
            class="text-white text-3xl font-Rajdhani"
          >
            Pasquale Palena Â© 2023
          </h3>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
